migrations-2/marketplace2.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE "users" ("id" bigserial,"email" text,"password" text,"name" text,"balance" decimal,"image_url" text,"created_at" timestamptz,"updated_at" timestamptz,"deleted_at" timestamptz);
+CREATE INDEX "idx_users_deleted_at" ON "users" ("deleted_at");
+CREATE INDEX "idx_users_id" ON "users" ("id");
+
+CREATE  TABLE Users_1 (
```

Comment:
Why not keep a consistent naming scheme? If GORM uses snake_case let's stick with this convention?

Also, why are there two spaces? :sweat_smile: